Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -595,7 +595,7 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
                                                                                 serverSelectionMethod: currentServerSelectionMethod,
                                                                                 includedRoutes: includedRoutes ?? [],
                                                                                 excludedRoutes: settings.excludedRanges,
-                                                                                regenerateKey: false)
+                                                                                regenerateKey: true)
```

Comment:
Always use a new key when starting up the tunnel.  Why not?

Reply:
One downside here is that it'll make the NetP connection speed overall feel slower since it has to do extra work to get started.

Reply:
I'm not sure... after all we're not saving a call to register right now.